Source/utils/ini.cpp

```diff
@@ -117,7 +117,7 @@ tl::expected<Ini, std::string> Ini::parse(std::string_view buffer)
 			nextLineBegin = eof;
 		} else {
 			nextLineBegin = lineEnd + 1;
-			if (lineBegin + 1 < lineEnd && *(lineEnd - 1) == '\r') --lineEnd;
+			if (lineBegin + 1 <= lineEnd && *(lineEnd - 1) == '\r') --lineEnd;

```

Comment:
doesn't seem related :)

Reply:
It's in a separate commit that I also sent as another PR